CHANGELOG.unreleased.md

```diff
@@ -40,15 +41,6 @@
 
 <!--packages-start-->
 
-- @vates/task patch
-- @xen-orchestra/backups patch
-- @xen-orchestra/log minor
-- @xen-orchestra/mixins feat
-- @xen-orchestra/xapi patch
-- vhd-lib patch
-- xo-cli patch
-- xo-server minor
-- xo-server-perf-alert patch
```

Comment:
This change shouldn't be there. It's probably because you're not up to date with master. You can rebase on master now or simply remove this change and rebase later.

Reply:
It's done!